### ISSUE TITLE
Separating sentences into lines creates breaks

### ISSUE THREAD

**Issue by aidan-mueller** [cobelu/Long-EZ#4]
The style guide says:

> Each sentence should be on it's own line.

I was in the process of separating some lines in a chapter, but decided to do a test. What I found is that separating sentences into lines creates mid-paragraph line breaks:

Sentence in paragraph with newlines between sentences.
Sentence in paragraph with newlines between sentences.
Sentence in paragraph with newlines between sentences.

Sentence in paragraph without newlines between sentences. Sentence in paragraph without newlines between sentences. Sentence in paragraph without newlines between sentences.

Is this intended behavior? It could significantly lengthen the rendered text (unless a custom renderer is used). It would also read a bit strangely.

That said, I do think some of the paragraphs in the plans could be broken up into smaller paragraphs.

**Comment by cobelu** [comment]
I'm sorry. This is a phrasing issue. 
There should be a line break after every sentence.
This does **not** imply an **empty line** between each sentence. 
That will cause a new paragraph each time.
See attached for example (note line numbers on left).

It's good to hit the enter key after every sentence to help readability during editing.
We do this at work for writing papers in LaTeX.
It helps a lot.

Large, rambling paragraphs can be split into two (or maybe even three) paragraphs.
They should be grouped by main ideas (as we do with paragraphs in writing).
That is, a large paragraph describing the installation of multiple parts, each with many sentences, should likely be split into multiple paragraphs.
<img width="949" alt="Screen Shot 2020-06-07 at 10 22 40 AM" src="https://user-images.githubusercontent.com/6185895/83973076-8c990780-a8a9-11ea-96bb-446b34817004.png">

Note that this is ok because it should not change the integrity or meaning of the words.

Best,
-C

**Comment by aidan-mueller** [comment]
I actually didn't find the style guide to be unclear. My concern is the fact that line breaks in the source text result in line breaks within a paragraph in the final result. In my example paragraphs above, you can see that the first paragraph with line breaks at the end of the sentence is longer since the text doesn't fill the width of the text area and wrap around.

I definitely agree it's easier to read/edit in the source text. I'm just concerned about the result.

**Comment by cobelu** [comment]
I see how "line breaks in the source text result in line breaks within a paragraph in the final result" as you describe, but I'm not able to replicate your results for display. When I edit (even on GH), I can't seem to replicate this issue. Please see attached screenshots.

1. Which editor are you using?
2. Which OS are you using?
3. Have you tried pushing and reviewing what is displayed on GH (go ahead and give it a shot)?

<img width="1552" alt="Screen Shot 2020-06-08 at 8 53 11 AM" src="https://user-images.githubusercontent.com/6185895/84038565-b879c300-a965-11ea-9009-77c9b29e67dd.png">

<img width="1552" alt="Screen Shot 2020-06-08 at 8 53 40 AM" src="https://user-images.githubusercontent.com/6185895/84038538-b283e200-a965-11ea-9848-29709ce8315e.png">

**Comment by aidan-mueller** [comment]
Looking at the file in the image, it is displaying the same for me as it is in the screenshot. This is not the case for the issue text editor. Although both use markdown, they have different behavior in this regard.

TL;DR this was me failing to test thoroughly.
